Docs/docs/worlds/components/vrc_station.md

```diff
@@ -51,11 +51,15 @@ The default `VRCChair` prefab included in the SDK can be used on avatars to let
 
 When using stations on an avatar that you want to animate on or off, you need to toggle specific objects and components. 
 ![image](/img/worlds/vrc_station-0adc923-av-station-fix.png)
-**Do not animate or disable the toggle in the red box. Only animate the toggles in the green boxes.** Since this involves disabling/enabling components and objects, this **must** be done in the FX layer.
+**The red box needs to be enabled upon upload. If it is disabled upon upload, the Station will not work!**
```

Comment:
Should "Station" be in lowercase?

```suggestion
**The red box (in the screenshot above) needs to be enabled upon upload. If it is disabled upon upload, the station will not work!**
```

Unrelated: Do you think adding "(in the screenshot above)" makes sense?

Reply:
Yeah, lets keep them lowercase for consistency, and good idea to make sure people arent looking for red boxes in the inspector